src/main/java/minicraft/core/Game.java

```diff
@@ -32,18 +33,29 @@ protected Game() {} // Can't instantiate the Game class.
 	public static int MAX_FPS;
 
 	// DISPLAY
-	static Display display = null, newDisplay = null;
-	public static void setDisplay(@Nullable Display display) { newDisplay = display; }
+	static Display curDisplay = null;
```

Comment:
If you are going to name a variable, it is better to use full words. So in this case it would be currentDisplay.